lib/checks/aria/unsupportedrole.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "unsupportedrole",
```

Comment:
Should this be hyphenated? `unsupported-role` is more ascetically pleasing and easier to read (IMO).

Reply:
Ah, makes sense. I'm +1 for consistency, so no need to change